### ISSUE TITLE
Error in Travis CI: Cannot find module '@octokit/request-error'

### ISSUE THREAD

**Issue by Kunduin** [semantic-release/semantic-release#1206]
## Current behavior

Use default npx semantic-release in Travis CI and failed.

## Environment

- **semantic-release** version: 15.13.16
- CI environment: Travis CI
- Plugins used: null
- **semantic-release** configuration: just default npx semantic-release
- CI logs: [https://travis-ci.com/Kunduin/react-picture-annotation/builds/115520396](https://travis-ci.com/Kunduin/react-picture-annotation/builds/115520396)

```bash
An error occurred while running semantic-release: { Error: Cannot find module '@octokit/request-error'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/travis/build/Kunduin/react-picture-annotation/node_modules/@octokit/request/dist-node/index.js:11:20)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module '@octokit/request-error'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/travis/build/Kunduin/react-picture-annotation/node_modules/@octokit/request/dist-node/index.js:11:20)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }Script failed with status 1
```

**Comment by 3dwardCh3nG** [comment]
I am having the same issue here which blocks the release. Can someone look into this and provide a solution/workaround as soon as possible?

**Comment by JakeGinnivan** [comment]
I think https://github.com/octokit/request-error.js/issues/8#issuecomment-501951044 is the source issue.

**Comment by zeke** [comment]
I just ran into this too. 👀 

- https://travis-ci.org/nice-registry/nice-package/builds/545565970
- https://github.com/nice-registry/nice-package/pull/24

**Comment by gr2m** [comment]
fixed via 1.0.4, sorry friends 👋
